completions-sample/src/extension.ts

```diff
@@ -27,35 +33,44 @@ export function activate(context: vscode.ExtensionContext) {
 			// be inserted and then the character will be typed.
 			const commitCharacterCompletion = new vscode.CompletionItem('console');
 			commitCharacterCompletion.commitCharacters = ['.'];
-			commitCharacterCompletion.documentation = new vscode.MarkdownString('Press `.` to get `console.`');
+			commitCharacterCompletion.documentation = new vscode.MarkdownString(
+				'Press `.` to get `console.`'
+			);
 
 			// a completion item that retriggers IntelliSense when being accepted,
-			// the `command`-property is set which the editor will execute after 
-			// completion has been inserted. Also, the `insertText` is set so that 
+			// the `command`-property is set which the editor will execute after
+			// completion has been inserted. Also, the `insertText` is set so that
 			// a space is inserted after `new`
 			const commandCompletion = new vscode.CompletionItem('new');
 			commandCompletion.kind = vscode.CompletionItemKind.Keyword;
 			commandCompletion.insertText = 'new ';
-			commandCompletion.command = { command: 'editor.action.triggerSuggest', title: 'Re-trigger completions...' };
+			commandCompletion.command = {
+				command: 'editor.action.triggerSuggest',
+				title: 'Re-trigger completions...',
+			};
 
 			// return all completion items as array
 			return [
 				simpleCompletion,
 				snippetCompletion,
 				commitCharacterCompletion,
-				commandCompletion
+				commandCompletion,
 			];
-		}
+		},
 	});
 
 	const provider2 = vscode.languages.registerCompletionItemProvider(
 		'plaintext',
 		{
-			provideCompletionItems(document: vscode.TextDocument, position: vscode.Position) {
-
+			provideCompletionItems(
+				document: vscode.TextDocument,
+				position: vscode.Position
+			) {
 				// get all text until the `position` and check if it reads `console.`
 				// and if so then complete if `log`, `warn`, and `error`
-				const linePrefix = document.lineAt(position).text.substr(0, position.character);
+				const linePrefix = document
+					.lineAt(position)
+					.text.slice(0, position.character);

```

Comment:
The `substr` to `slice` refactor is the only code change. All the other changes are formatting done automatically by Prettier.

Reply:
> All the other changes are formatting done automatically by Prettier

This codebase doesn't use prettier. Please try to revert the formatting changes

Reply:
Reverted all formatting changes ✅